resources/views/element/header.blade.php

```diff
@@ -0,0 +1,178 @@
+<header class="header">
+    <div class="under-header"></div>
+    <div class="container">
+        <div class="header-main d-none d-sm-none d-md-block d-lg-block">
+            <div class="row">
+                <div class="col-sm-2">
+                    <div class="logo">
+                        <a href="/" class="logo-wrapper ">
+                            <img src="{{ asset('frontend/images/logo/logo.png') }}" alt="logo Luz Bakery">
```

Comment:
```suggestion
                            <img src="{{ asset('frontend/images/logo/logo.png') }}" alt="__('something')">
```
use https://laravel.com/docs/5.8/localization

Reply:
em đang làm giao diện, làm logic e sẽ làm multi language

Reply:
i18n luôn nhé